src/components/Garden/DisputeFees.js

```diff
@@ -50,9 +50,9 @@ function DisputeFees({ proposal }) {
             <div>
```

Comment:
Let´s maybe conditionally render this div if `proposal.challengerArbitratorFee` exists?